flutter/lib/src/integrations/widgets_binding_integration.dart

```diff
@@ -10,10 +9,12 @@ import '../widgets_binding_observer.dart';
 ///   - [SentryWidgetsBindingObserver]
 ///   - [WidgetsBindingObserver](https://api.flutter.dev/flutter/widgets/WidgetsBindingObserver-class.html)
 class WidgetsBindingIntegration extends Integration<SentryFlutterOptions> {
-  SentryWidgetsBindingObserver? _observer;
+  late SentryWidgetsBindingObserver _observer;
+  late SentryFlutterOptions _options;
```

Comment:
Better to keep it nullable, so if `close` is called before `call` which is wrong, wouldn't throw, it's defensive programming against API misusing.